src/views/forms/FormAccountRestrictionTransaction/FormAccountRestrictionTransaction.vue

```diff
@@ -8,7 +8,10 @@
 
                     <RestrictionDirectionInput v-model="formItems.direction" :disabled="isDeleteMode || directionDisabled" />
 
-                    <RestrictionTypeInput v-model="formItems.blockType" :disabled="isDeleteMode || transactionTypeDisabled" />
+                    <RestrictionTypeInput
+                        v-model="formItems.blockType"
+                        :disabled="isDeleteMode || transactionTypeDisabled || restrictionTxType === 'TRANSACTION_TYPE'"
```

Comment:
Should we move `restrictionTxType === 'TRANSACTION_TYPE'` into `transactionTypeDisabled` method ?